src/main/java/uk/gov/hmcts/reform/idam/web/helper/ErrorHelper.java

```diff
@@ -27,4 +36,25 @@ public static void showError(String errorTitle, String errorMessage, String erro
         model.put(ERROR_LABEL_ONE, errorLabelOne);
         model.put(ERROR_LABEL_TWO, errorLabelTwo);
     }
+
+    public static HttpStatusCodeException restException(@Nullable String message,
+                                                        HttpStatus status, HttpHeaders headers,
+                                                        String error, String description) {
+        return restException(message, status, headers, jsonErrorString(error, description).getBytes());
+    }
+
+    public static HttpStatusCodeException restException(
+        @Nullable String message, HttpStatus status, HttpHeaders headers, byte[] body) {
+        if (status.series() == HttpStatus.Series.CLIENT_ERROR) {
+            return HttpClientErrorException.create(message, status, status.getReasonPhrase(), headers, body, UTF_8);
+        }
+        return HttpServerErrorException.create(message, status, status.getReasonPhrase(), headers, body, UTF_8);
+    }
+
+    public static String jsonErrorString(String error, String description) {
+        return "{\n" +
+            "  \"error\": \"errorString\",\n" +
+            "  \"description\": \"errorDescription\"\n" +
+            "}";
```

Comment:
Should be returning values from eror and description variables:
```suggestion
        return String.format("{%n \"error\": \"%s\",%n  \"description\": \"%s\"%n}", error, description);
```

Reply:
lol that was the idea but im dumb